gen/dcos-config.yaml

```diff
@@ -4,6 +4,7 @@ package:
       {
         "environment": {
           "MESOS_IP_DISCOVERY_COMMAND": "/opt/mesosphere/bin/detect_ip",
+          "PLATFORM": "{{ platform }}"
```

Comment:
https://teamcity.mesosphere.io/viewLog.html?buildId=389831&buildTypeId=DcosIo_Dcos_VagrantIntegrationTests_ProvisioningTestGenconfBashPr&tab=buildLog#_state=1508,1514&focus=2600 -- missing comma after this